lib/diagnostic.js

```diff
@@ -8,22 +8,32 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
-
+  let result = Math.min();
```

Comment:
Did you look up the definition/syntax for `Math.min()`? Never copy code blindly. Just Google "Javascript math.min()". 

Please study this solution:
```
const min = function min() {
  let result = arguments[0];
  for (let i = 1; i < arguments.length; i++) {
    if (arguments[i] < result) {
      result = arguments[i];
    }
  }

  return result;
};
```